### PR TITLE
Feature/euf 300 doc new menu

### PR DESCRIPTION
**Pull request by brunocavaliere**

### CODE REVIEW

**Review by edgardleal — APPROVED** [review]
It sounds good to me, but please check out the comments I left and see if they make sense.

File: sdk-doc/content/posts/hello-world.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+title: Hello, World!
```
Comment (LucasEstreladeOliveira) [inline sdk-doc/content/posts/hello-world.md:2]: Do we need this one?

File: sdk-doc/docusaurus.config.js [hunk 2]
```diff
@@ -33,18 +33,47 @@ const config = {
   },
 
   presets: [
+    // [
```
Comment (LucasEstreladeOliveira) [inline sdk-doc/docusaurus.config.js:36]: Could we just remove the lines and not comment them? 🤔

**Review by LucasEstreladeOliveira — CHANGES_REQUESTED** [review]

**Review by LucasEstreladeOliveira — APPROVED** [review]
lgtm

**Pull request closed without merging**